src/Faker/Provider/fa_IR/Color.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Faker\Provider;
+
+/**
+ * @author lsv
+ */
+class Color extends Base
```

Comment:
Please extend `\Faker\Provider\Color`, that way you don't have to repeat most methods.
And have a look at the [color provider from ar_SA](https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/ar_SA/Color.php), he included a source as well.